### PR TITLE
Create Transaction

### PR DESCRIPTION
**Pull request by junzai97**
Code fix for creating transaction

### CODE REVIEW

File: src/test/java/com/thirdcc/webapp/web/rest/TransactionResourceIT.java [hunk 1]
```diff
@@ -518,6 +519,9 @@ void getAllTransactionsByLastModifiedByNotContainsSomething() throws Exception {
      * Executes the search, and checks that the default entity is returned.
      */
     private void defaultTransactionShouldBeFound(String filter) throws Exception {
+        User currentUser = getCurrentUser();
+        String currentUserName = currentUser.getFirstName() + " " + currentUser.getLastName();
+        System.out.println(transaction);
```
Comment (HowardYaw) [inline src/test/java/com/thirdcc/webapp/web/rest/TransactionResourceIT.java]: why sout?

File: src/test/java/com/thirdcc/webapp/web/rest/TransactionResourceIT.java [hunk 2]
```diff
@@ -285,6 +285,7 @@ void getAllTransactionsByEventIdIsEqualToSomething() throws Exception {
     void getAllTransactionsByEventIdIsNotEqualToSomething() throws Exception {
         // Initialize the database
         transactionRepository.saveAndFlush(transaction);
+        System.out.println("ALERT: " + transaction.getCreatedBy());
```
Comment (HowardYaw) [inline src/test/java/com/thirdcc/webapp/web/rest/TransactionResourceIT.java]: why sout?

File: src/test/java/com/thirdcc/webapp/web/rest/TransactionResourceIT.java (hunk 1 above)
Comment (junzai97) [reply src/test/java/com/thirdcc/webapp/web/rest/TransactionResourceIT.java]: I have forgot to removed it. Was using it for debug.
Resolved in 3626026d5c8a71fc2d94b86caff93c2d0353dbd5

File: src/test/java/com/thirdcc/webapp/web/rest/TransactionResourceIT.java (hunk 2 above)
Comment (junzai97) [reply src/test/java/com/thirdcc/webapp/web/rest/TransactionResourceIT.java]: Resolved in 3626026d5c8a71fc2d94b86caff93c2d0353dbd5

**Review by HowardYaw — APPROVED** [review]

**Review by CarlosSia — APPROVED** [review]

**Pull request merged**